config/show_config

```diff
@@ -1,162 +1,131 @@
-  config_message="$config_message\n - FLOAT:\t\t\t\t $TARGET_FLOAT"
-  config_message="$config_message\n - FPU:\t\t\t\t\t $TARGET_FPU"
-  config_message="$config_message\n - CPU features:\t\t\t $TARGET_FEATURES"
-  config_message="$config_message\n - LTO (Link Time Optimization) support: $LTO_SUPPORT"
-  config_message="$config_message\n - GOLD (Google Linker) Support:\t $GOLD_SUPPORT"
-  config_message="$config_message\n - LLVM support:\t\t\t $LLVM_SUPPORT"
-  config_message="$config_message\n - DEBUG:\t\t\t\t ${DEBUG:-no}"
-
-  # config_message="$config_message\n - CFLAGS:\t $TARGET_CFLAGS"
-  # config_message="$config_message\n - LDFLAGS:\t $TARGET_LDFLAGS"
-
-  # Graphic configuration
-
-  config_message="$config_message\n\n Graphic configuration:"
-  config_message="$config_message\n $dashes$dashes"
-
-  if [ "$DISPLAYSERVER" = "x11" ] ; then
-    config_message="$config_message\n - Xorg Graphic Drivers:\t\t $GRAPHIC_DRIVERS"
-    config_message="$config_message\n - XORG Composite support:\t\t $COMPOSITE_SUPPORT"
-    config_message="$config_message\n - WindowManager:\t\t\t $WINDOWMANAGER"
-  fi
-  config_message="$config_message\n - OpenGL (GLX) support (provider):\t $OPENGL_SUPPORT ($OPENGL)"
-  config_message="$config_message\n - OpenGLES support (provider):\t\t $OPENGLES_SUPPORT ($OPENGLES)"
+  config_message+="\n\n Buildsystem configuration:"
+  config_message+="\n $dashes$dashes"
 
-  # Hardware decoder support
+  config_message+="\n - CPU (ARCH):\t\t\t\t $TARGET_CPU ($TARGET_ARCH)"
+  config_message+="\n - FLOAT:\t\t\t\t $TARGET_FLOAT"
+  config_message+="\n - FPU:\t\t\t\t\t $TARGET_FPU"
+  config_message+="\n - CPU features:\t\t\t $TARGET_FEATURES"
+  config_message+="\n - LTO (Link Time Optimization) support: $LTO_SUPPORT"
+  config_message+="\n - GOLD (Google Linker) Support:\t $GOLD_SUPPORT"
+  config_message+="\n - LLVM support:\t\t\t $LLVM_SUPPORT"
+  config_message+="\n - DEBUG:\t\t\t\t ${DEBUG:-no}"
 
-  config_message="$config_message\n\n Hardware decoder configuration:"
-  config_message="$config_message\n $dashes$dashes"
-
-  config_message="$config_message\n - Kodi Player driver:\t\t\t $KODIPLAYER_DRIVER"
-  config_message="$config_message\n - VAAPI Support:\t\t\t $VAAPI_SUPPORT"
-  config_message="$config_message\n - VDPAU Support:\t\t\t $VDPAU_SUPPORT"
-
-  # Input device configuration
-
-  config_message="$config_message\n\n Input device configuration:"
-  config_message="$config_message\n $dashes$dashes"
-
-  config_message="$config_message\n - Remote support:\t\t\t $REMOTE_SUPPORT"
-  config_message="$config_message\n - CEC Adapter support:\t\t\t $CEC_SUPPORT"
-  config_message="$config_message\n - CEC Framework support:\t\t $CEC_FRAMEWORK_SUPPORT"
-  config_message="$config_message\n - Kodi Joystick support:\t\t $JOYSTICK_SUPPORT"
+  # config_message+="\n - CFLAGS:\t $TARGET_CFLAGS"
+  # config_message+="\n - LDFLAGS:\t $TARGET_LDFLAGS"
```

Comment:
intentional out-commented ?

Reply:
These lines are commented in the current show_config. Happy to make the lines active again.

Reply:
![image](https://user-images.githubusercontent.com/1355173/83358001-7d2d3200-a370-11ea-96a4-3e85a33b6595.png)
I guess nuke it

Reply:
Went ahead and uncommented.

Reply:
I guess we misunderstood each other :) The lines are out commented since at least 8 years so we can remove them.

Reply:
Uncommenting led to #4406, and would have been helpful in earlier buildsystem changes. Has consideration been made on adding a $build.config that outputs this script to put with the image, tar and sha256 values?